### PR TITLE
add awslogs check

### PR DESCRIPTION
**Pull request by cnelson**
This is current deployed via concourse and aims to emulate the current riemann configuration

### CODE REVIEW

File: ci/awslogs.sh [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+# in seconds
+ALERT_THRESHOLD=3600
+STOP_THRESHOLD=5400
+
+# monitor these log grouips
+cat <<EOF > /tmp/monitor_loggroups
+kubernetes-development
+kubernetes-production
+kubernetes-staging
+this-log-group-will-alert-cnelson
```
Comment (jmcarp) [inline ci/awslogs.sh]: Is it useful to keep something like this around for testing, or should we drop it?

File: bosh/alerts/awslogs.alerts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+ALERT AWSLogsGroupExpired
```
Comment (jmcarp) [inline bosh/alerts/awslogs.alerts]: WDYT about emitting a timestamp of the last successful log group check and alerting if too much time has passed?

File: ci/awslogs.sh (hunk 1 above)
Comment (cnelson) [reply ci/awslogs.sh]: I'll delete it before we merge, I just left it in for testing so I could validate this was working

File: bosh/alerts/awslogs.alerts (hunk 2 above)
Comment (cnelson) [reply bosh/alerts/awslogs.alerts]: ✔️

**Pull request merged**